case-lib/opt.sh

```diff
@@ -3,28 +3,28 @@
 # Before using this, you must define these option arrays in your test
 # script. They must all be indexed by some unique, one-character
 # codename for each of your option.
-declare -A OPT_DESC OPT_NAME OPT_PARM_lst OPT_VALUE_lst
+declare -A OPT_DESC OPT_NAME OPT_HAS_ARG OPT_ARG
 
 # option setup && parse function
 func_opt_parse_option()
 {
     # OPT_NAME     (long) option name
     # OPT_DESC    short sentence describing the option
-    # OPT_PARM_lst    0 or 1: number of argument required
-    # OPT_VALUE_lst   default value overwritten by command line
+    # OPT_HAS_ARG    0 or 1: number of argument required
+    # OPT_ARG   default value overwritten by command line
     #                 input if any. Set to 0 or 1 when PARM=0
```

Comment:
```suggestion
    #                 input if any. Set to 0 or 1 when OPT_HAS_ARG    =0
```

And try to refine the text here to make it more easy to understand

Reply:
Updated